setup.py

```diff
@@ -38,13 +38,13 @@
     packages=find_packages(),
     package_data={'ifaddr': ['py.typed']},
     license='MIT',
+    python_requires=">=3.8",
     classifiers=[
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Topic :: System :: Networking',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.7',
```

Comment:
Please restore this line, it's not related to the purpose of this PR

Reply:
I mean, we *can*... but I disagree -- I see the scope of the PR as "bring packaging metadata in line with stated supported versions of Python." If it'd be better for me to update the PR title, I'd be happy to do that instead.

Reply:
That'll work